src/audio/host-zephyr.c

```diff
@@ -243,35 +243,19 @@ void host_common_update(struct host_data *hd, struct comp_dev *dev, uint32_t byt
 {
 	struct comp_buffer __sparse_cache *source;
 	struct comp_buffer __sparse_cache *sink;
-	int ret;
```

Comment:
Looks like from commit message this should be 2 patches ? i.e. stage 1 and 2 above seem to be independent ?